### PR TITLE
Improve input Search

### PR DESCRIPTION
**Pull request by ntmedina**
Context: We don't need to use blue color (our interactive color) in the search icon because it isn't an interactive one, it exists just for semantic purposes.

- Fix search icons (Color and shapes)
___

Antes
screen shot 2018-09-14 at 3 17 47 pm

___

Depois
screen shot 2018-09-14 at 3 16 15 pm

### CODE REVIEW

File: react/components/InputSearch/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import Input from '../Input'
 import SearchIcon from '../icon/Search'
-import DenyIcon from '../icon/Deny'
+import FailureIcon from '../icon/Failure'
```
Comment (lbebber) [inline react/components/InputSearch/index.js]: 👀 acho que deveriamos renomear isso (ou criar um alias)

**Comment by gabrielgalc** [comment]
Looks better to me!

**Comment by cmdalbem** [comment]
The component had been originally designed for the cenario where a submit action is needed, as documented in the original issue: https://github.com/vtex/styleguide/issues/144. In Credit Control we couldn't have a "dynamic search" behavior for technical reasons.
I suggest we review together this design to see how we can make it work for both cenarios. Please don't merge this PR since it'll effect projects currently using it.

**Comment by felippenardi** [comment]
@ntmedina Is this still going to be merged?

**Comment by cmdalbem** [comment]
We discussed this in a design meeting a couple of weeks ago and decided a redesign was needed. I created an issue to address these: https://github.com/vtex/styleguide/issues/423

**Pull request closed without merging**